contracts/proposal/SoftwareUpgradeProposal.sol

```diff
@@ -36,6 +44,11 @@ contract SoftwareUpgradeProposal is DelegatecallExecutableProposal, Cancelable {
 
     function executeDelegateCall(address selfAddr, uint256) external override {
         SoftwareUpgradeProposal self = SoftwareUpgradeProposal(selfAddr);
```

Comment:
Add explanation why self() is needed, like:
```
SoftwareUpgradeProposal self = SoftwareUpgradeProposal(selfAddr); // needed because this runs in Governance contract context
```